src/main/java/org/osiam/AuthServer.java

```diff
@@ -0,0 +1,89 @@
+package org.osiam;
+
+import javax.servlet.*;
+import javax.sql.*;
+
+import org.flywaydb.core.*;
+import org.flywaydb.core.api.*;
+import org.springframework.beans.factory.annotation.*;
+import org.springframework.boot.*;
+import org.springframework.boot.autoconfigure.*;
+import org.springframework.boot.builder.*;
+import org.springframework.boot.context.web.*;
+import org.springframework.context.annotation.*;
+import org.springframework.security.authentication.encoding.ShaPasswordEncoder;
+import org.springframework.transaction.annotation.*;
+import org.springframework.web.filter.*;
+
+import com.zaxxer.hikari.*;
+
+@SpringBootApplication
+@EnableTransactionManagement
+@EnableAspectJAutoProxy(proxyTargetClass = true)
+@PropertySource("classpath:/auth-server.properties")
+@ImportResource("classpath:/spring-servlet.xml")
+public class AuthServer extends SpringBootServletInitializer {
+
+    @Value("${org.osiam.auth-server.db.driver}")
+    private String driverClassName;
+
+    @Value("${org.osiam.auth-server.db.url}")
+    private String databaseUrl;
+
+    @Value("${org.osiam.auth-server.db.username}")
+    private String databaseUserName;
+
+    @Value("${org.osiam.auth-server.db.password}")
+    private String databasePassword;
+
+    @Value("${org.osiam.auth-server.db.vendor}")
+    private String databaseVendor;
+
+    public static void main(String[] args) {
+        SpringApplication.run(AuthServer.class, args);
+    }
+
+    @Override
+    protected SpringApplicationBuilder configure(SpringApplicationBuilder application) {
```

Comment:
Ah ok. So both is working, the fat war start with java -jar and deployment in a servlet container?